sessions/js-math-and-numbers/counter/js/index.js

```diff
@@ -0,0 +1,9 @@
+/*
+Exercise: Implement a simple counter
+
+- [ ] Add an event listener for each of the two buttons
+- [ ] Increments/decrements the value stored in the counter variable on click.
+- [ ] Show the value in the <p> element.
+*/
+
+// let counter = 0;
```

Comment:
Here we could also argue about adding the queries. 
```suggestion
/*
Exercise: Implement a simple counter

- [ ] Add an event listener for each of the two buttons
- [ ] Increments/decrements the value stored in the counter variable on click.
- [ ] The counter should be shown in the result element.
*/

let counter = 0;
```

Reply:
Okay. I added the queries